### PR TITLE
Added support for a user whitelist in idlemove

### PR DESCRIPTION
**Pull request by MatthewLavine**
I've added the ability to define a list of users which are exempt from being moved by this module. This was useful for me since I have a few bots on my server which do not talk often.

### CODE REVIEW

File: modules-available/idlemove.ini [hunk 1]
```diff
@@ -35,4 +36,4 @@ channel = 0
 ;threshold = 60
 ;mute = True
 ;deafen = False
-;channel = 1
```
Comment (hacst) [inline modules-available/idlemove.ini]: Whitespace change

File: modules/idlemove.py [hunk 2]
```diff
@@ -57,7 +58,8 @@ class idlemove(MumoModule):
                              ('mute', commaSeperatedBool, [True]),
                              ('deafen', commaSeperatedBool, [False]),
                              ('channel', commaSeperatedIntegers, [1]),
-                             ('source_channel', commaSeperatedIntegers, [-1])
+                             ('source_channel', commaSeperatedIntegers, [-1]),
+                             ('whitelist', commaSeperatedStrings, [''])
```
Comment (hacst) [inline modules/idlemove.py]: Should be ```[]```, shouldn't it?

File: modules/idlemove.py [hunk 3]
```diff
@@ -220,4 +226,4 @@ def stopped(self, server, context = None):
         sid = server.id()
         self.affectedusers[sid] = set()
         self.log().debug('Server %d gone', sid)
-    
\ No newline at end of file
+
```
Comment (hacst) [inline modules/idlemove.py]: Whitespace change

**Comment by hacst** [comment]
Sounds useful. See my comments above and I noticed some lines have trailing spaces. Apart from that looks good to me.

**Comment by MatthewLavine** [comment]
Thanks for checking out this pull request!

I've corrected the issues you mentioned in modules/idlemove.py, namely the string initialization and the trailing line at the end of the file.

I don't see exactly what you mentioned in modules-available/idlemove.ini, I see no extra whitespace at the end of the file.

**Comment by hacst** [comment]
Merged as e9ea8cd Added support for a user whitelist in idlemove .
Thanks. Took me long enough to rediscover this. Sorry.

**Pull request closed without merging**